config/environments/production.rb

```diff
@@ -98,5 +98,5 @@
   config.x.policy_disregards_start_date = Date.parse(ENV['POLICY_DISREGARDS_START_DATE'] || '2021-1-8')
 
   # says whether or not to send alerts to Sentry and support email for undeliverable emails
-  config.x.alert_undeliverable_emails = HostEnv.production?
+  config.x.alert_undeliverable_emails = true
```

Comment:
you can probably remove this line now as what is in `app/services/govuk_emails/email_monitor.rb` only checks the values of this for development or test